src/uu/df/src/table.rs

```diff
@@ -144,23 +144,24 @@ impl From<Filesystem> for Row {
             ffree,
             ..
         } = fs.usage;
+        let bused = blocks - bfree;
         Self {
             fs_device: dev_name,
             fs_type,
             fs_mount: mount_dir,
             bytes: blocksize * blocks,
-            bytes_used: blocksize * (blocks - bfree),
+            bytes_used: blocksize * bused,
             bytes_avail: blocksize * bavail,
             bytes_usage: if blocks == 0 {
                 None
             } else {
-                Some(((blocks - bfree) as f64) / blocks as f64)
+                Some((bused as f64) / (bused + bavail) as f64)
```

Comment:
Based on your description, this seems reasonable, but I would not have been able to describe why this change is important without it. Could you please leave a comment in the code here explaining why we should use this formula to compute the usage percentage?

Reply:
Makes sense, I added a corresponding comment.